packages/connect-display/src/rpc.ts

```diff
@@ -0,0 +1,151 @@
+import { createClient, createServer } from '@ceramicnetwork/rpc-window'
```

Comment:
Is this a new file or a moved one please? I thought we already had this logic somewhere.

Reply:
I was confused on the first run through of this and thought I needed to update Connect-display. I've reverted it all back to what is in develop so no longer an issue.